micrometer-core/src/main/java/io/micrometer/core/instrument/binder/system/FileDescriptorMetrics.java

```diff
@@ -27,20 +27,39 @@
 import java.lang.management.OperatingSystemMXBean;
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.List;
 
 import static java.util.Collections.emptyList;
 
 /**
- * File descriptor metrics.
+ * File descriptor metrics gathered by the JVM.
+ * <p>
+ * Supported JVM implementations:
+ * <ul>
+ *     <li>HotSpot</li>
+ *     <li>J9</li>
+ * </ul>
  *
  * @author Michael Weirauch
+ * @author Tommy Ludwig
```

Comment:
I'm not sure the conventions for `@author` tags in the project. For instance, `ProcessorMetrics` doesn't have any `@author` tags, but this class did, so I added my name.